content/k8s/fluxcd/helmrelease/_index.md

```diff
@@ -134,14 +142,24 @@ resources:
   - cert-manager
 ```
 
+And last but not least we need to add the values.yaml
+```
+# infrastructure/cert-manager/values.yaml
```

Comment:
Shouldn't this be _before_ adding the `ClusterIssuer`, to install the cert-manager with the CRDs?